installer/arch/prepare

```diff
@@ -0,0 +1,244 @@
+#!/bin/sh -e
+# Copyright (c) 2014 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+PKGEXT='tar.xz'
+DISTROAKA='arch'
+
+# aurcompile <package> <extra makepkg options>: compile package from AUR.
```

Comment:
- `touch`: touchegg and its dependencies (frame, grail, geis). The alternative to AUR is to drop support (like we do with Debian)
 - `chrome-common`: I'll look into it. The PKGBUILD does a number of things, though, including pulling in dependencies...
 - `audio`: `lib32-sbc` (which I maintain, and is actually out of date... I wonder if we could get it merged into `[multilib]`)